repo/meta/schema/package-schema.json

```diff
@@ -58,6 +58,10 @@
       "type": "string",
       "description": "Post installation notes that would be useful to the user of this package."
     },
+    "postUninstallNotes": {
+      "type": "string",
+      "description": "Post uninstall notes that would be useful to the user of this package."
```

Comment:
Please s/uninstall/uninstallation for consistency with `preInstallNotes` and `postInstallNotes` descriptions.